ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -64,6 +64,7 @@ const (
 	tcAddQdiscRootCommandString      = "tc qdisc add dev %s root handle 1: prio priomap 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2"
 	tcAddQdiscLatencyCommandString   = "tc qdisc add dev %s parent 1:1 handle 10: netem delay %dms %dms"
 	tcAddQdiscLossCommandString      = "tc qdisc add dev %s parent 1:1 handle 10: netem loss %d%%"
+	tcAllowlistIPCommandString       = "tc filter add dev %s protocol ip parent 1:0 prio 1 u32 match ip dst %s flowid 1:3"
 	tcAddFilterForIPCommandString    = "tc filter add dev %s protocol ip parent 1:0 prio 1 u32 match ip dst %s flowid 1:1"
```

Comment:
Let's use prio (priority) of 2 for IP in the disallow list and use prio of 1 for IP in the allow list. Because the allowlist takes precedence.

Reply:
> The protocol field describes protocol that the filter will be applied to. We will only discuss case of ip protocol. The preference field (priority can be used alternatively) sets the priority of currently defined filter. This is important, since you can have several filters (lists of rules) with different priorities. Each list will be passed in the order the rules were added, then list with lower priority (higher preference number) will be processed. The parent field defines the CBQ tree top (e.g. 1:0), the filter should be attached to.

https://tldp.org/HOWTO/Adv-Routing-HOWTO/lartc.adv-filter.u32.html

Reply:
and I think we may need to remove the prio 2 handler - https://github.com/aws/amazon-ecs-agent/blob/dev/ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go#L1348-L1360

Reply:
I think we should just delete the root handle. Here's what happens when we try to delete a filter that doesn't exist:
```
# Filter that doesn't exist:
% tc filter del dev eth0 prio 2
Error: Filter with specified priority/protocol not found.

# Filter that exists:
% tc filter del dev eth0 prio 1
# No output as expected
```